authzed/api/v1/experimental_service.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+package authzed.api.v1;
+
+option go_package = "github.com/authzed/authzed-go/proto/authzed/api/v1";
+option java_package = "com.authzed.api.v1";
+
+import "google/api/annotations.proto";
+import "validate/validate.proto";
+
+import "authzed/api/v1/core.proto";
+
+// ExperimentalService exposes a number of APIs that are currently being
+// prototyped and tested for future inclusion in the stable API.
+service ExperimentalService {
+
+  // BulkLoadRelationships is a faster path to writing a large number of
+  // relationships at once. It is both batched and streaming. For maximum
+  // performance, the caller should attempt to write relationships in as close
+  // to relationship sort order as possible: (resource.object_type,
+  // resource.object_id, relation, subject.object.object_type,
+  // subject.object.object_id, subject.optional_relation)
+  // 
+  // EXPERIMENTAL
+  rpc BulkLoadRelationships(stream BulkLoadRelationshipsRequest)
```

Comment:
I think it might be worth a requirement that any experimental API has a referenced Github issue with it, placed here